### DISCUSSION TITLE
How can I copy and exit?

### DISCUSSION THREAD

**Discussion by rew1nter** [hluk/CopyQ#2660] [Q&A] [answered]
Is there a way to copy the selected item and exit the window with the same key press?

**Comment by hluk** [comment]
Do you mean something like "Close main window" option (in History tab in Preference dialog)?

If you mean exit CopyQ completely after copying something, that may not be possible since the CopyQ itself must provides the clipboard content. That means if you copy something in CopyQ and exit the app, the clipboard content would be cleared.

**Reply by rew1nter** [reply]
I mean to select something from here and also close this window after selection, upon pressing enter 
![image](https://github.com/hluk/CopyQ/assets/64508791/0a8ddbcc-2fbf-47e0-8a0c-1f68c883d886)

**Reply by hluk** [reply]
That's handled by the "Close main window" option.

**Reply by rew1nter** [reply]
Could you elaborate a bit more on how I go about achieving this?
![image](https://github.com/hluk/CopyQ/assets/64508791/9f5c0f54-3e4d-4b35-b16f-74066f356071)
I want this shortcut and would also like to close the main window upon pressing enter.
Here I'm trying to mimic KDE which does the same thing

**Reply by hluk** [reply, marked as answer]
Remove the Return/Enter shortcut assigned to "Copy Selected Items" action and enable the options "Close main window" and "Paste to current window" (also under History tab) in Preferences dialog. This will make the Enter key copy the selected items and close the main window.

![image](https://github.com/hluk/CopyQ/assets/38054/62914e82-6c40-443d-8b04-3c76cc70bb43)

**Reply by rew1nter** [reply]
This works like a charm. Thank you

**Reply by tmccombs** [reply]
That wasn't enough for me. I also had to enable the "hide main window" option in the Layout options. I am using a tiling window manager.
